packages/webdriverio/tests/commands/browser/newWindow.test.ts

```diff
@@ -108,4 +108,36 @@ describe('newWindow', () => {
         }).catch((err: Error) => err) as Error
         expect(error.message).toContain('not supported on mobile')
     })
+
+    it('should open a new window by default', async () => {
+        const browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+
+        const newHandle = await browser.newWindow('https://webdriver.io', {
+            windowName: 'some window'
+        })
+
+        expect(newHandle.type).toBe('window')
+    })
+
+    it('should open a new tab when type is tab', async () => {
+        const browser = await remote({
+            baseUrl: 'http://foobar.com',
+            capabilities: {
+                browserName: 'foobar'
+            }
+        })
+
+        const newHandle = await browser.newWindow('https://webdriver.io', {
+            type: 'tab',
+            windowName: 'some tab'
+        })
+
+        expect(newHandle.type).toBe('tab')
+    })
```

Comment:
Can we update these and actually check if the bidi command is called correctly? In order to run a mocked Bidi session the `browserName` needs to be called `bidi` and we have to spy on some commands on the `browser` object. For inspiration, please take a look at the `switchTab` command.

Reply:
Added tests as suggested!